README.md

```diff
@@ -7,7 +7,7 @@
 [![Security Vulnerabilities](https://img.shields.io/github/issues/opensearch-project/OpenSearch/security%20vulnerability?labelColor=red)](https://github.com/opensearch-project/OpenSearch/issues?q=is%3Aissue+is%3Aopen+label%3A"security%20vulnerability")
 [![Open Issues](https://img.shields.io/github/issues/opensearch-project/OpenSearch)](https://github.com/opensearch-project/OpenSearch/issues)
 [![Open Pull Requests](https://img.shields.io/github/issues-pr/opensearch-project/OpenSearch)](https://github.com/opensearch-project/OpenSearch/pulls)
-[![2.8 Open Issues](https://img.shields.io/github/issues/opensearch-project/OpenSearch/v2.8.0)](https://github.com/opensearch-project/OpenSearch/issues?q=is%3Aissue+is%3Aopen+label%3A"v2.8.0")
+[![2.9 Open Issues](https://img.shields.io/github/issues/opensearch-project/OpenSearch/v2.9.0)](https://github.com/opensearch-project/OpenSearch/issues?q=is%3Aissue+is%3Aopen+label%3A"v2.9.0")
```

Comment:
Should this be 2.10? Is the intent here to show the open issues going into the next minor release?

Reply:
Since 2.9 hasn't released yet :)

Reply:
It will be soon. 
Why not just add this to our version increment bot?

Reply:
The increment bot does a bunch of file specific regex magic. We will have to write some good additions to the script for it. 
I cut a new issue here: https://github.com/opensearch-project/OpenSearch/issues/8851

Reply:
Updated to 2.10

Reply:
Thanks, this should be fairly simple for the bot.